oxcaml/tests/simd/stubs.c

```diff
@@ -94,6 +95,12 @@ static inline simd_float32x4_t simd_float32x4_round_near(simd_float32x4_t v)
   return vrndnq_f32(v);
 }
 
+static inline simd_int32x4_t simd_float64x2_to_int32x2_trunc(simd_float64x2_t v)
+{
+  int32x2_t i32 = vqmovn_s64(vcvtq_s64_f64(v));
+  return vcombine_s32(i32, i32);
```

Comment:
should the high part be 0?
```suggestion
  return vcombine_s32(i32,0);
```